### ISSUE TITLE
Deliverable: Mid-term release plan

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#84]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Mid-term release plan done and uploaded as defined in homework/deliverables word document

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Finished in 8b9cdf974b0592bb76a4d87a3d35fe9834c38272
